.travis.yml

```diff
@@ -37,7 +40,7 @@ install:
         || echo dev-${TRAVIS_BRANCH}) \
         travis_retry composer update $COMPOSER_FLAGS --prefer-dist --no-interaction;
     fi
-  - ./phpcq update
+  - ./phpcq update -t ./.phpcq
```

Comment:
Why this change? When doing so, it must also get changed below in the run statement as otherwise the tools can't be located anymore.

Reply:
I wanted to enable vendor caching again by using a custom phpcq directory. Applied the the option to the run command as well.